user/travis-pro.md

```diff
@@ -61,3 +61,16 @@ Travis CI is, and always will be, free for open source projects.
 
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
+
+## Making a private repository public
+
+If you have a private repository that you'd like to make public, first change
+the setting on GitHub, then resync your Travis CI account:
+
+1. Follow the [instructions on
+GitHub](https://help.github.com/articles/making-a-private-repository-public/) on
```

Comment:
Here, we also need to mention about changing the notify url.

But sometimes this doesn't work either, as we realized in the recent mails. So please add a line, that if these steps don't work, they should write to us